packages/gatsby/cache-dir/static-entry.js

```diff
@@ -246,20 +211,17 @@ export default (pagePath, callback) => {
     pathPrefix: __PATH_PREFIX__,
   })
 
-  scripts
-    .slice(0)
-    .reverse()
-    .forEach(script => {
-      // Add preload/prefetch <link>s for scripts.
-      headComponents.push(
-        <link
-          as="script"
-          rel={script.rel}
-          key={script.name}
-          href={`${__PATH_PREFIX__}/${script.name}`}
-        />
-      )
-    })
+  scripts.forEach(script => {
+    // Add preload/prefetch <link>s for scripts.
+    headComponents.push(
+      <link
+        as="script"
+        rel={script.rel}
+        key={script.file}
+        href={`${__PATH_PREFIX__}/${script.file}`}
```

Comment:
Does `script.file` === `script.name` here? Why the switch?

Reply:
I'm presuming it's because you changed the algo in `assetsForCurrentPage()` so it's returning a slightly different structure?

Reply:
I moved some of the work that was done here to the stats plugin, which creates an asset object like: `{ file, chunkName, rel }`, file felt more descriptive i guess esp, next to the added "chunkName".